tests/app/services/test_request_supplementary_data.py

```diff
@@ -47,6 +56,27 @@
     },
 }
 
+encrypted_mock_supplementary_data_payload = {
+    "dataset_id": "44f1b432-9421-49e5-bd26-e63e18a30b69",
```

Comment:
since we have a fair few bits of json now, it might be nice to pull these out into a `conftest.py` file for the services folder and keep the tests cleaner looking